contents/teams/developer-experience/objectives.md

```diff
@@ -0,0 +1,12 @@
+## Q1 2026 Objectives
+
+- Enable the creation of **modern, fully isolated products** with CI tests completing in approximately **5 minutes**
+  - Establish Bazel and a scalable, well-defined product structure
+- Clarify and standardize **type usage**, including clear boundaries and expectations for product isolation
+- Evolve **DevEx engineering best practices**, particularly around product setup, isolation, and maintainability and brief all engineers regularly
+- Continuously improve **CI/CD reliability and quality**
```

Comment:
with a good dashboard template in PostHog you could productise this for other teams